CONTRIBUTING.md

```diff
@@ -1298,35 +1299,16 @@ There are two possible choices for which commit to use:
 2. Checkout commit `C`, the hypothetical result of what would happen if the PR
    were merged into `master` (automatically generated by GitHub).
 
-This choice depends on several factors; here is the decision tree as of
-2021-03-30:
-
-- For CI jobs on CircleCI:
-  - If the name of the job (or one of its ancestors in the workflow DAG)
-    contains "xla" or "gcc5", choice **2** is used. This includes the following
-    jobs:
-    - pytorch_linux_xenial_py3_6_gcc5_4_build
-      - pytorch_cpp_doc_build
-      - pytorch_doc_test
-      - pytorch_linux_forward_backward_compatibility_check_test
-      - pytorch_linux_xenial_py3_6_gcc5_4_jit_legacy_test
-      - pytorch_linux_xenial_py3_6_gcc5_4_test
-      - pytorch_python_doc_build
-    - pytorch_xla_linux_bionic_py3_6_clang9_build
-      - pytorch_xla_linux_bionic_py3_6_clang9_test
-  - Otherwise, choice **1** is used.
-- For CI jobs on GitHub Actions:
-  - If the PR was created using [`ghstack`](https://github.com/ezyang/ghstack),
-    choice **1** is used.
-  - Otherwise, choice **2** is used.
-
-This is important to be aware of, because if you see a CI failure on your PR and
-choice **2** is being used for that CI job, it is possible that the failure is
-nondeterministically caused by a commit that does not exist in the ancestry of
-your PR branch. If you happen to have write access to this repo, you can choose
-to use `ghstack` to eliminate this nondeterminism for GitHub Actions jobs on
-your PRs, but it will still be present for the select CircleCI jobs listed
-above.
+For all practical purposes, most people can think of the commit being used as 
+commit `B` (choice **1**).
```

Comment:
This is wrong for the stacked PRs or for PR that do not target master. I.e. A is not `refs/head/master` but commit `refs/pull/42/base` belongs to I guess

Reply:
I've reworded the chart a bit to use more generic terms for the source commit and the branch the PR would be merged into.

Does this make it more accurate?